docs/components/base/sensor-controlled.md

```diff
@@ -82,3 +83,7 @@ The following base control API methods are available with feedback control on a
 
 - [SetVelocity()](/components/base/#setvelocity): available if base is configured to receive angular and linear velocity feedback.
 - [Spin()](/components/base/#spin): available if base is configured to receive orientation feedback.
```

Comment:
Relatedly, @martha-johnston , I realized I'm actually confused what happens if the user specifies multiple movement sensors in the config. Does their output get merged somehow? When would a user want to do that rather than use a merged movement sensor?

Reply:
it will go through all the sensors and return the first sensor that supports the property it's looking for. so first go through all the sensors and find one that supports orientation, then go through them all again and find one that supports linear and angular velocity

Reply:
got it, makes sense!

Reply:
no need to change this for this PR, but is there a reason to support this functionality separate from merged movement sensor? if not i could file a ticket for the RDK backlog to make this a string field

Reply:
gotcha. i'll go ahead and file a ticket so that we can have a discussion about it with rand